File: Level_1/problem_1188.java
import java.util.Scanner;

public class Problem_1188 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        String op = sc.nextLine().trim();

        double[][] matrix = new double[12][12];

        for (int i = 0; i < 12; i++) {
            for (int j = 0; j < 12; j++) {
                matrix[i][j] = sc.nextDouble();
            }
        }

        double totalSum = 0.0;
        int count = 0;

        for (int i = 7; i < 12; i++) {
            for (int j = 12 - i; j < i; j++) {
                totalSum += matrix[i][j];
                count++;
            }
        }

        double result = 0.0;
        if (op.equals("S")) {
            result = totalSum;
        } else if (op.equals("M")) {
            result = totalSum / count;
        }

        System.out.printf("%.1f\n", result);
    }
}

File: Level_1/problem_1188.py
op = input().strip()

matrix = []
for i in range(12):
    row = []
    for j in range(12):
        row.append(float(input().strip()))
    matrix.append(row)

total_sum = 0.0
count = 0

for i in range(7, 12): 
    for j in range(12 - i, i): 
        total_sum += matrix[i][j]
        count += 1

if op == 'S':
    result = total_sum
elif op == 'M':
    result = total_sum / count

print(f"{result:.1f}")
